test/videojs-contrib-hls.test.js

```diff
@@ -2780,7 +2780,7 @@ QUnit.test('cleans up the buffer when loading VOD segments', function(assert) {
   this.clock.tick(1);
   this.player.currentTime(120);
   this.player.tech_.hls.mediaSource.sourceBuffers[0].trigger('updateend');
-  this.clock.tick(1);
+  this.clock.tick(2);
```

Comment:
This would be good to add as a comment above the clock tick (since clock ticks are generally pretty mysterious through our tests).